tests/yast2_gui/yast2_firewall.pm

```diff
@@ -123,10 +123,11 @@ sub change_interface_zone {
 sub verify_zone {
     my (%args) = @_;
 
-    my $interfaces = $args{interfaces} //= 'no_interfaces';
-    my $default    = $args{default}    //= 'no_default';
+    my $interfaces    = $args{interfaces}    //= 'no_interfaces';
+    my $default       = $args{default}       //= 'no_default';
+    my $menu_selected = $args{menu_selected} //= 0;
 
-    assert_and_click 'yast2_firewall_zones';
+    assert_and_click 'yast2_firewall_zones' unless $menu_selected;
```

Comment:
would a comment be useful here or above in the definition of the variable? wdyt?

Reply:
It is obviously referring to 'zone' menu according to the needle that is matching or not depending on the flag, I always prefer to create variable with meaningful name than write comments when the code does not have already those comments all over the place. Of course 'obviously' is arguable :) because I did most of the original code and I might be not so objective, so it probably a matter or taste for me that I don't like to see too much comments if not needed because you can infer it from the code itself.